community/Makefile

```diff
@@ -9,8 +9,8 @@ default: \
 reset-docs:
 	git checkout HEAD -- ./sig-list.md ./sig-*/README.md ./wg-*/README.md
 
-generate:
-	podman run --rm -e WHAT -e GO111MODULE -e GOPROXY -v $(shell pwd):/go/src/app:Z $(GOLANG_IMAGE) make -C /go/src/app generate
+generate-sig-files:
```

Comment:
With that target name change, please also do the same change in line 7 (changing the default from `generate` to `generate-sig-files`